app/views/directives/annotations.html

```diff
@@ -1,10 +1,10 @@
-<p>
+<p ng-class="{'mar-bottom-xl': !expandAnnotations}">
```

Comment:
`p` have 10.5px bottom margin, but in this case, since almost all annotations appear last, I give these 20px to match the bottom margin of what most other elements have `.table, dl, forms`.